Sources/SwiftKuery/ConnectionPool.swift

```diff
@@ -98,8 +142,15 @@ public class ConnectionPool {
         give(connection)
     }
     
-    /// Release all the connections in the pool by calling connectionReleaser closure on each connection,
-    /// and empty the pool.
+    // MARK: Disconnect Pool
+    /**
+     Release all the connections in the pool by calling `connectionReleaser` closure on each connection, and empty the pool.
```

Comment:
Can we reword this to:
"Removes all the connections from the pool and calls the `connectionReleaser` closure on each."